config/locales/de.yml

```diff
@@ -438,17 +921,147 @@ de:
+    error_no_event: |
+      Diese Konferenz hat noch keine Einreichungen.
+
+      Sie können Events im Reiter 'Events' hinzufügen oder einen Call for
+      Participation starten, damit Teilnehmer Events einreichen können.
+    error_no_room: |
+      Bevor Sie Events anlegen können, müssen Sie für die Konferenz mindestens einen Raum
+      eingerichtet haben. Dies können Sie im Reiter <em> Einstellungen </ em> tun.
+    event_attachments: Hochgeladene Dateien
+    event_classifiers: Classifier
+    event_details: Event-Details
+    event_feedbacks: Event-Feedbacks
+    event_ratings: Event-Bewertungen der Reviewer
+    events_by_gender: Events nach Geschlecht
+    events_by_language: Events nach Sprache
+    events_by_state: Events nach Status
+    events_in_current_conference: Events in der aktuellen Konferenz
+    events_in_other_conferences: Events in anderen Konferenzen
+    feedback_not_enabled: Das Feedback-System ist nicht aktiviert.
+    hide_all: Alles verbergen
+    inputs:
+      hints:
+        do_not_record: Soll dieses Event aufgezeichnet werden?
+        note: 'Sichtbar nur für Administratoren.'
+        public: Ist dieses Event im öffentlichen Programm sichtbar?
+        recording_license: Aufzeichnungs-Lizenz für dieses Event
+        submission_note: 'Sichtbar für Administratoren und Benutzer; zusätzliche Information.'
+        tech_rider: Was wird für diese Veranstaltung benötigt?
+    my_events: Meine Events
+    my_rating: Meine Bewertungen
+    my_ratings: Meine Bewertungen
+    no_attende_more_infos: |
+      Wenn Sie das Teilnehmer-Feedback für die Konferenz unter aktivieren
+      (oben im Reiter Einstellungen), wird der statische Export des Programms Links zum
+      Feedback-Formular enthalten. Dieses Feedback wird hier angezeigt.
+    no_attendee: Für dieses Event gibt es noch kein Feedback.
+    no_events: |
+      Diese Konferenz hat noch keine Einreichungen.
+
+      Sie können Events im Reiter 'Events' hinzufügen oder einen Call for
+      Participation starten, damit Teilnehmer Events einreichen können.
+    no_ratings: |
+      Bisher hat noch niemand eine Bewertung für dieses Event abgegeben. Seien Sie der erste
+      und bewerten Sie das Event mit dem obigen Formular.
+    not_involved: |
+      Sie sind für diese Konferenz in keinem Event involviert. Wenn Sie ein Event einreichen
+      oder einem beitreten, taucht er in dieser Liste auf.
+    print_accepted_cards: Druckkarten (nur accepted)
+    print_accepted_cards_hint: |
+      Laden Sie ein PDF mit nur den Event im Zustand accepted als Karten herunter (vier Karten pro Seite).
+      Besonders nützlich, um Ihr Programm offline zu planen.
+    print_cards: Karten drucken
+    print_cards_hint: |
+      Laden Sie ein PDF mit allen Events als Karten herunter (vier Karten pro Seite).
+      Besonders nützlich, um Ihren Programm offline zu planen.
+    public_attachment_hint: Ist dieses Attachment öffentlich?
```

Comment:
The texts before always simply named the attachments "Datei", so this would be "Ist diese Datei öffentlich". But in this case it might be better to say "Is dieser Anhang öffentlich".